redisearch/query.go

```diff
@@ -297,6 +297,20 @@ func (q *Query) SetReturnFields(fields ...string) *Query {
 	return q
 }
 
+// AddReturnFields adds the fields that should be returned from each result
+// to the ReturnFields property
+func (q *Query) AddReturnFields(fields ...string) *Query {
```

Comment:
Take a look at this for great doc foo. https://github.com/googleapis/googleapis/blob/master/google/api/http.proto